grid-proxy/internal/explorer/converters.go

```diff
@@ -59,6 +58,7 @@ func nodeFromDBNode(info db.Node) types.Node {
 		ExtraFee:          info.ExtraFee,
 	}
 	node.Status = nodestatus.DecideNodeStatus(node.Power, node.UpdatedAt)
+	node.Dedicated = info.FarmDedicated || info.NodeContractCount == 0 || info.RentContractID != 0
```

Comment:
I am curious on why having no node contracts make the node dedicated? are newly registered nodes with no contracts considered dedicated or am I missing dedicated meaning? :)

Reply:
At first you needed to mark your farms as dedicated to make your nodes available to rent. but this changed and if there are any node that is completely free and have no contracts it can marked as dedicated and be available to rent.
so yes newly registered nodes will be dedicated

Reply:
so dedicated means available to be rented? but that means rented nodes (third condition) are not available to be rented, right?

Reply:
dedicated should include both:
- `rentable`: nodes in dedicated farms, or empty nodes.
- `rented`: nodes already rented. (it have/don't have node-contracts. anyway it will be considered as dedicated)